wagtail/admin/templates/wagtailadmin/shared/button.stories.tsx

```diff
@@ -104,6 +104,32 @@ const Template = ({ url }) => (
       button disabled
     </button>
 
+    <h4>Bi-color Secondary icon buttons with text</h4>
```

Comment:
Not huge - but we probably do not need `Secondary` with the upper case s here.
This title is title case, which means only the first word has the upper case letter, unless the words inside are proper nouns, `secondary` in this context is not a proper noun.

Reply:
Thanks a lot @lb- ! Hope this is a first of many amazing contributions to come...